std/engine/face/wasm_sim_face.go

```diff
@@ -14,21 +14,29 @@ import (
 type WasmSimFace struct {
```

Comment:
Do we still need WASM build? It seems that no one has ever used it.

Reply:
I'm fixing it right now ;)